http-server/src/middlewares/proxy_request.rs

```diff
@@ -0,0 +1,174 @@
+//! Middleware that proxies requests at a specified URI to internal
```

Comment:
Name bikeshedding; `ProxyRequest` feels a bit vague to me.

How about something like `GetRequestToRpc` or `ProxyGetRequest` just for a touch more detail?

Reply:
ProxyGetRequest is a better name for it

Reply:
I'm sold for `ProxyGetRequest` does eliminate any confusion! Thanks!